spec/ruby/core/process/spawn_spec.rb

```diff
@@ -26,7 +26,7 @@
       code = "STDERR.puts 'hello'"
       cmd = "Process.wait Process.spawn(#{ruby_cmd(code).inspect}, #{@options.inspect})"
       ruby_exe(cmd, args: "2> #{@output}")
-      File.binread(@output).should == "hello#{newline}"
+      File.binread(@output).should =~ /hello#{newline}/
```

Comment:
What warning?
The system ruby should not be used, `ruby_cmd` runs the mspec Ruby.